docs/release/trg-0/trg-1-06.md

```diff
@@ -0,0 +1,30 @@
+---
+title: TRG 1.06 - Administrator's Guide (User Assistance)
+sidebar_position: 1
+---
+
+:::caution
+Proposed release date: 15th of January 2024
+:::
+
+| Status     | Created      | Post-History                           |
+|------------|--------------|----------------------------------------|
+| Draft      | 31-Jan-2024  | created initial Draft                  |
+
+## Why
+
+Administrator's Guide is intended for administrators that install and operate the solution.
```

Comment:
I would re-word it, so that the reason why we need an admin guide is better described.
Something like "Tractus-X components and applications are operated in various productive environments. An Administrators guide can help operators in setting up and maintaining these environments. It is also a vehicle share best-practices..."

Reply:
Tractus-X components and applications may be operated in various productive environments. An Administrator`s guide shall help operators to install and operate the solution. It can also be a vehicle to share best-practices on maintenance etc.